src/Simplex/Messaging/Agent/Store/SQLite.hs

```diff
@@ -3017,6 +3023,26 @@ deleteDeletedSndChunkReplicasExpired db ttl = do
   cutoffTs <- addUTCTime (-ttl) <$> getCurrentTime
   DB.execute db "DELETE FROM deleted_snd_chunk_replicas WHERE created_at < ?" (Only cutoffTs)
 
+updateServersStats :: DB.Connection -> AgentPersistedServerStats -> IO ()
+updateServersStats db stats = do
+  updatedAt <- getCurrentTime
+  DB.execute db "UPDATE servers_stats SET servers_stats = ?, updated_at = ? WHERE servers_stats_id = 1" (stats, updatedAt)
+
+getServersStats :: DB.Connection -> IO (Either StoreError (Maybe AgentPersistedServerStats))
+getServersStats db =
+  firstRow fromOnly SEServersStatsNotFound $
+    DB.query_ db "SELECT servers_stats FROM servers_stats WHERE servers_stats_id = 1"
+
+getServersStatsStartedAt :: DB.Connection -> IO (Either StoreError UTCTime)
```

Comment:
why does it need to be a separate query and not together with get?

Also, I don't see INSERT that would set started_at